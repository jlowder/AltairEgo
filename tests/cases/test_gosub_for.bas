10 LET N=2
20 LET A=1
30 LET B=2
40 LET C=3
120 FOR D=1 TO N : PRINT : PRINT "TRIAL #";D; : INPUT X,Y,Z
130 IF ABS(X-A)+ABS(Y-B)+ABS(Z-C)=0 THEN 300
140 GOSUB 500 : PRINT : NEXT D
150 END
300 PRINT "YOU GOT IT!"
310 END
500 PRINT "IN SUBROUTINE"
510 RETURN
