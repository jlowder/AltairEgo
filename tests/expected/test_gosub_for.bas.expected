Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK

TRIAL # 1 ? IN SUBROUTINE


TRIAL # 2 ? IN SUBROUTINE

OK
